pkg/util/maps/maps.go

```diff
@@ -18,21 +18,13 @@ limitations under the License.
 // when `x/exp/maps` graduates to stable.
 package maps
 
-// Clone clones the input preserving the difference between nil and empty
-func Clone[K comparable, V any, S ~map[K]V](s S) S {
-	if s == nil {
-		return nil
-	}
-
-	if len(s) == 0 {
-		return S{}
-	}
+import (
+	"maps"
+)
 
-	ret := make(S, len(s))
-	for k, v := range s {
-		ret[k] = v
-	}
-	return ret
+// Clone uses clone std built-in function
+func Clone[K comparable, V any, S ~map[K]V](s S) S {
+	return maps.Clone(s)
 }
```

Comment:
we can also go with an alias, but I think is best just drop this implementation and use the std one in the whole cod-base

Reply:
+1 on dropping